snuba/admin/static/api_client.tsx

```diff
@@ -57,6 +64,25 @@ function Client() {
           })
       );
     },
+    getClickhouseCannedQueries: () => {
+      const url = baseUrl + "clickhouse_queries";
+      return fetch(url).then((resp) => resp.json());
+    },
+    executeQuery: (query: QueryRequest) => {
+      const url = baseUrl + "run_clickhouse_system_query";
+      return fetch(
+        new Request(url, {
+          method: "POST",
+          body: new Blob([JSON.stringify(query)], { type: "application/json" }),
```

Comment:
Why blob? Why not just post the JSON?

Reply:
I just copied this example from MDN: https://developer.mozilla.org/en-US/docs/Web/API/Request/blob#examples

Reply:
I think we should just send JSON like the rest of the API
```suggestion
return fetch(
  headers: { "Content-Type": "application/json" },
  url,
  method: "POST",
  body: JSON.stringify(query)
)
```